backend/CONTRIBUTING.md

```diff
@@ -3,7 +3,7 @@
 ## Install and run locally
 
 ```
-docker-compose run backend
+docker-compose up backend
```

Comment:
Is that still true @philipdickinson ?

Reply:
Yeah, the `local` override file just maps 8081 to 80, so doesn't affect the backend.